common/postgresql-ng/templates/secrets.yaml

```diff
@@ -16,20 +16,26 @@ spec:
       - name: channel-config
         # at the time of writing, kubectl is only available in edge
         image: "{{ $.Values.global.dockerHubMirror }}/library/alpine:edge"
+        {{- $users := "" }}
+        {{- range $user, $settings := .Values.users }}
+        {{- $users = printf "%s postgres-user-password-%s" $users $user}}
```

Comment:
These secret names should contain `.Release.Name` to avoid conflicts between multiple Postgres DBs within the same namespace.